tests/test-amp-img-sanitizer.php

```diff
@@ -128,6 +128,16 @@ public function get_data() {
 <img src="http://placehold.it/380x180" width="380" height="180" />',
 				'<amp-img src="http://placehold.it/350x150" width="350" height="150" class="amp-wp-enforced-sizes" layout="intrinsic"></amp-img><amp-img src="http://placehold.it/360x160" width="360" height="160" class="amp-wp-enforced-sizes" layout="intrinsic"></amp-img><amp-img src="http://placehold.it/370x170" width="370" height="170" class="amp-wp-enforced-sizes" layout="intrinsic"></amp-img><amp-img src="http://placehold.it/380x180" width="380" height="180" class="amp-wp-enforced-sizes" layout="intrinsic"></amp-img>',
 			),
+
+			'image_center_aligned'                     => array(
+				'<img class="aligncenter" src="http://placehold.it/350x150" width="350" height="150" />',
+				'<figure class="aligncenter" style="max-width: 350px;"><amp-img class="amp-wp-enforced-sizes" src="http://placehold.it/350x150" width="350" height="150" layout="intrinsic"></amp-img></figure>',
+			),
+
+			'image_left_aligned'                       => array(
+				'<img class="alignleft" src="http://placehold.it/350x150" width="350" height="150" />',
+				'<amp-img class="alignleft amp-wp-enforced-sizes" src="http://placehold.it/350x150" width="350" height="150" layout="intrinsic"></amp-img>',
+			),
```

Comment:
Should there be a test for when an image has a caption? Or is that unnecessary because in that case the `figure` will have the class name instead of the `img`?

Reply:
Yes, good idea. [This commit](https://github.com/Automattic/amp-wp/pull/1109/commits/79958f52660ff435832530a0ce1f77046a1d3a00) adds a test for an image with a caption. It ensures that `<figure>` elements don't have the class `aligncenter` stripped, only `<amp-img>` elements.